test/fixtures/azure-fixture/src/main/java/fixture/azure/AzureHttpHandler.java

```diff
@@ -264,7 +275,100 @@ public void handle(final HttpExchange exchange) throws IOException {
                 exchange.sendResponseHeaders(RestStatus.OK.getStatus(), response.length);
                 exchange.getResponseBody().write(response);
 
+            } else if (Regex.simpleMatch("POST /" + account + "/" + container + "*restype=container*comp=batch*", request)) {
+                // Blob Batch (https://learn.microsoft.com/en-us/rest/api/storageservices/blob-batch)
+                final StringBuilder response = new StringBuilder();
+
+                try (BufferedReader requestReader = new BufferedReader(new InputStreamReader(exchange.getRequestBody()))) {
+                    String batchBoundary = requestReader.readLine();
+                    String responseBoundary = "batch_" + UUID.randomUUID();
+
+                    String line;
+                    String contentId = null, requestId = null, toDelete = null;
+                    while ((line = requestReader.readLine()) != null) {
+                        if (batchBoundary.equals(line) || (batchBoundary + "--").equals(line)) {
+                            // Found the end of a single request, process it
+                            if (contentId == null || requestId == null || toDelete == null) {
+                                throw new IllegalStateException(
+                                    "Missing contentId/requestId/toDelete: " + contentId + "/" + requestId + "/" + toDelete
+                                );
+                            }
+
+                            // Process the deletion
+                            if (blobs.remove("/" + account + toDelete) != null) {
+                                final String acceptedPart = Strings.format("""
+                                    --%s
+                                    Content-Type: application/http
+                                    Content-ID: %s
+
+                                    HTTP/1.1 202 Accepted
+                                    x-ms-delete-type-permanent: true
+                                    x-ms-request-id: %s
+                                    x-ms-version: 2018-11-09
+
+                                    """, responseBoundary, contentId, requestId).replaceAll("\n", "\r\n");
```

Comment:
This uses fixed `"\r\n"` instead of `System.lineSeparator()` because the server is meant to follow windows convention?

Reply:
The azure client expects line breaks to be windows style. I checked and that's not a HTTP requirement. Cheeky Microsoft.